workspaces/liferay-dlfoldertemplate-workspace/client-extensions/liferay-dlfoldertemplate-etc-node/util/config-util.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import config from './configTreePath.js';
+
+export function getConfigByKey(configKey) {
+	configKey = configKey.toString().toUpperCase().replaceAll('.', '_');
```

Comment:
It looks like you're declaring all the values of configKey in your constant files, so it seems like you shouldn't need to have so much logic for changing the format. Can this be simplified?